issue_template.md

```diff
@@ -0,0 +1,12 @@
+### Steps to reproduce
```

Comment:
I borrowed this issue template from [rails](https://github.com/rails/rails/blob/master/.github/issue_template.md).